tests/core/environment-file/test.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+. /usr/share/beakerlib/beakerlib.sh || exit 1
+
+rlJournalStart
+    rlPhaseStartTest "Run pytest session"
+        rlRun "pytest -svv"
```

Comment:
I liked this approach, as I was able to test everything locally. Also, I don't need to have any knowledge of this beaker tests syntax (this is a python project!!!)